_store/derpilzcc-readonly-access-bundle.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Readonly access plugin for Kimai 2
+name: Readonly access plugin
+intro: "A plugin, which allows to create readonly users, that can read times recorded for a special company."
+developer: derpilzcc
+date: "2019-08-09 20:00:00 +0200"
+icon: fas fa-mouse-pointer
+price: "0"
+download: https://github.com/fungus75/ReadOnlyAccessBundle/archive/master.zip
+github: https://github.com/fungus75/ReadOnlyAccessBundle
+screenshot: 
+  - https://raw.githubusercontent.com/fungus75/ReadOnlyAccessBundle/master/screenshot.jpg
+tags:
+  - plugin
+---
+
+This bundle adds a special admin screen where the role can be assigned to an existing Kimai user.
```

Comment:
Can you explain the plugin a bit more, I don't understand what it does from that sentence.